restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -1981,11 +2016,19 @@ private void outboundToClient(final CreateCall request, final ActorRef sender) t
         }
     }
 
+    private SipURI addCustomHeadersForToUri (String customHeaders, SipURI to) {
```

Comment:
i think this is a bit tricky with many assumptions. I worry about runtimeExceptions like arrayindexoutofbounds, if passed string is not as expected, and then create leaks... Some additional checkings to make it safer and fail safe is appropiate would be better..

Reply:
Actually, checking if null for `customHeader` has been done before calling the method. 

I will add an extra check for the custom header that should contain `=`